### ISSUE TITLE
ProcessTimedOutException on large 7Zip archives

### ISSUE THREAD

**Issue by lasselehtinen** [wapmorgan/UnifiedArchive#10]
**Describe the bug**
I am hitting the ProcessTimedOutException on large 7Zip archive. This only occurs when processing files towards the end of the archive. For example the timeout is hit on file 28 out of 35.

**Configration (please complete the following information):**
 - OS: Linux
 - Version 0.1.0

**To Reproduce**
```
$archive = UnifiedArchive::open('path/to/large/7zip-archive.7z');
$archive->getFileContent($filename);
```

**Expected behavior**
The process timeout should be configurable. I noticed you are using Gemorroj/Archive7z for the this, so I am not sure the best way to handle this. I guess first that package should start supporting the timeout?

```
[Symfony\Component\Process\Exception\ProcessTimedOutException]
The process "'/usr/bin/7za' 'x' 'largeArchive.7z' '-so' 'output.ext' '-y' '-p '" exceeded the timeout of 60 seconds.


Exception trace:
 Symfony\Component\Process\Process->checkTimeout() at /app/vendor/symfony/process/Process.php:383
 Symfony\Component\Process\Process->wait() at /app/vendor/symfony/process/Process.php:202
 Symfony\Component\Process\Process->run() at /app/vendor/symfony/process/Process.php:222
 Symfony\Component\Process\Process->mustRun() at /app/vendor/gemorroj/archive7z/src/Archive7z.php:408
 Archive7z\Archive7z->getContent() at /app/vendor/gemorroj/archive7z/src/Entry.php:133
 Archive7z\Entry->getContent() at /app/vendor/wapmorgan/unified-archive/src/UnifiedArchive.php:554
 wapmorgan\UnifiedArchive\UnifiedArchive->getFileContent() at /app/app/Console/Commands/Archive.php:173
```

**Comment by wapmorgan** [comment]
May be support for configurable time-out will be added in future, but not now.

**Comment by wapmorgan** [comment]
Released 0.1.2 with 7zip-related fix. Now it has no timeout at all.

**Comment by wapmorgan** [comment]
@lasselehtinen Please, check out new release.
